index.html

```diff
@@ -0,0 +1,15 @@
+<!doctype html>
+<html>
+<head>
+  <meta charset="utf-8" />
+
+  <title>Netlify Jekyll CMS</title>
```

Comment:
Including "Jekyll" here could be confusing for some, this isn't really tied to a build tool.